### ISSUE TITLE
First and last names of bibtex in v1.8.0

### ISSUE THREAD

**Issue by byzheng** [kookma/TW-Refnotes#26]
The v1.8.0 can correctly parse author names with First and last name in bibtext, e.g.  `D.O. Borio and N.S. Schbib`. The reference in the text is rendered as `(Borio and Schbib, 2018)`

However, better bibtex in zotero exports as `Last Name, First Name` (e.g. `Borio, D.O. and Schbib, N.S.`. The reference in the text is rendered as `(D. and N., 2018)`. 

Could you support the new format in the authors?

**Comment by kookma** [comment]
This is difficult and make code complicated. We need to decide on standard format for storing bibtex data in a tiddler.
As ScienceDirect is one of the largest online database, I used its format as basis!

Do you have any suggestion? It seems there are several different formats used by different databases.

**Comment by byzheng** [comment]
Using space as splitter will cause confusing for names with multiple spaces, e.g. Isidro y Sánchez, Julio; van Eeuwijk, Fred A. 

The First Last format will not correct in this case, e.g. Julia Isidro y Sánchez into Sánchez, J. I. Y.; Fred A van Eeuwijk into Eeuwijk, F. A. V.

Last, First will be more accurate. 

I don't see ScienceDirect use First and Last format for names.

```
Scopus
EXPORT DATE: 01 June 2022

@ARTICLE{Yin2004426,
	author = {Yin, Xinyou and Struik, Paul C. and Kropff, Martin J.},
	title = {Role of crop physiology in predicting gene-to-phenotype relationships},
	year = {2004},
	journal = {Trends in Plant Science},
	volume = {9},
	number = {9},
	pages = {426 – 432},
	doi = {10.1016/j.tplants.2004.07.007},
	publication_stage = {Final},
	note = {Cited by: 178}
}
```

**Comment by kookma** [comment]
> I don't see ScienceDirect use First and Last format for names.

Export citation for https://www.sciencedirect.com/science/article/abs/pii/S0927775722007956

It will create

```
@article{ROSTAMI2022129040,
title = {Parametric study and optimization of oil drop process in a co-flowing minichannel},
journal = {Colloids and Surfaces A: Physicochemical and Engineering Aspects},
volume = {647},
pages = {129040},
year = {2022},
issn = {0927-7757},
doi = {https://doi.org/10.1016/j.colsurfa.2022.129040},
url = {https://www.sciencedirect.com/science/article/pii/S0927775722007956},
author = {Foruzan Rostami and Mohammad Rahmani},
keywords = {Droplet formation, Co-flow, Volume of fluid method, Two-phase flow, Response surface methodology}
}
```

**Comment by kookma** [comment]
I welcome a PR, if you think both situations can be handled!

**Comment by kookma** [comment]
Solved in TW 1.8.1
